src/components/@core/navigation-menu/light/join-group/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { Button } from "@chakra-ui/core";
+import useTranslation from "@configs/i18n/useTranslation";
+import { axCheckUserGroupMember, axJoinOpenUserGroup } from "@services/usergroup.service";
+import notification, { NotificationType } from "@utils/notification";
+import { useStoreState } from "easy-peasy";
+import React, { useEffect, useState } from "react";
+
+export default function JoinUserGroup() {
+  const { currentGroup, user } = useStoreState((s) => s);
+  const [isMember, setIsMember] = useState(false);
+  const { t } = useTranslation();
+
+  const addUserGroupMember = async () => {
+    const canAddMember = !currentGroup.isParticipatory
+      ? confirm(t("GROUP.CONFIRM_CLOSED_GROUP"))
+      : true;
+    if (canAddMember) {
+      const { success } = await axJoinOpenUserGroup(currentGroup.id);
+      if (success) {
+        setIsMember(true);
+        notification(
+          currentGroup.isParticipatory ? t("GROUP.MEMBER_JOINED") : t("GROUP.MEMBER_REQUESTED"),
+          NotificationType.Success
+        );
+      } else {
+        notification(t("MEMBER_JOINED_ERROR"), NotificationType.Error);
+      }
+    }
+  };
+
+  useEffect(() => {
+    (async () => {
+      const { data } = await axCheckUserGroupMember(currentGroup.id, user.id);
+      setIsMember(data);
+    })();
+  }, [currentGroup.name]);
+
+  return !isMember ? (
+    <Button variant="outline" variantColor="black" onClick={addUserGroupMember}>
+      {t("GROUP.JOIN")}{" "}
+    </Button>
+  ) : null;
```

Comment:
Reverse the condition so no `!` required

Reply:
also Can you take care of below points as well

- Outline button isn't visible use below kinda style
- also just use margin-left on button instead of applying `min-width` to logo because that will cause problems in XS responsive layout
- also buton should go to bottom for XS devices
![image](https://user-images.githubusercontent.com/5774849/86360153-971fa480-bc8f-11ea-8b72-00f6ff29d374.png)

```tsx
<Button
  variant="solid"
  size="sm"
  ml={{ base: 0, md: 4 }}
  variantColor="blue"
  onClick={addUserGroupMember}
  leftIcon="add"
>
  {t("GROUP.JOIN")}
</Button>
```

Reply:
Having `isLoading` configured for button will be nice so button can be automatically disabled and will prevent duplicate requests